fontforgeexe/fontview.c

```diff
@@ -245,180 +245,268 @@ extern BDFFont *FVSplineFontPieceMeal(SplineFont *sf, int layer, int ptsize, int
     return new;
 }
 
-static void FVDrawGlyph(GWindow pixmap, FontView *fv, int index, int request_expose ) {
-    GRect box, old2;
-    int feat_gid;
-    SplineChar *sc;
-    struct _GImage base;
-    GImage gi;
-    GClut clut;
-    int i,j;
-    int em = fv->b.sf->ascent+fv->b.sf->descent;
-    int yorg = fv->magnify*(fv->show->ascent);
+void FVDrawOutlineOnly(GWindow pixmap, SplinePointList *set,
```

Comment:
Spaces vs tabs garble the diff. Please adjust indents (tab stands for double indent, 4 spaces for single indent where present), at least in FVDrawGlyph().